lib/systems/include/TorqueControllerMux.h

```diff
@@ -76,9 +78,28 @@ class TorqueControllerMux
                                                     DrivetrainCommand_s previous_controller_command,
                                                     DrivetrainCommand_s desired_controller_out);
 
+    /// @brief Clamps negative rpms to 0f
+    /// @param const DrivetrainCommand_s &command provides the rpm info as a DrivetrainCommand_s
+    /// @return DrivetrainCommand_s
     DrivetrainCommand_s apply_positive_speed_limit_(const DrivetrainCommand_s &command);
+
+    /// @brief Ensure torque is at most at the specified limit. If exceeding, then limit it in the returned DrivetrainCommand_s
```

Comment:
You generally don't write "is" in the @param tags.

Reply:
Just put @param float max_torque The maximum average torque, in <units>, that the wheels may experience.